src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -41,29 +41,32 @@
 import org.xembly.Directives;
 
 /**
- * Starts Amazon EC2 instance.
+ * Starts Amazon ECS container, that runs Docker + SSHD on ECS. Container
+ * is configured in privileged mode. Configuration is stored in S3.
+ * See details here:
+ * @link http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html#ecs-config-s3
  * @author Yuriy Alevohin (alevohin@mail.ru)
  * @version $Id$
  * @since 2.0
- * @todo #629 Implement com.rultor.agents.ec2.StopsEC2 agent. It must
- *  stopped EC2 on-demand instance if it was started at StartsEC2 agent.
- *  StopsEC2 must use instance id from /talk/ec2/[@id] to stop it.
+ * @todo #629 Implement com.rultor.agents.ecs.StopsECS agent. It must
+ *  stop ECS on-demand container if it was started at StartsECS agent.
+ *  StopsECS must use container ARN from /talk/ecs/[@id] to stop it.
```

Comment:
@xupyprmv this needs to stay at `/talk/ec2` too, because of the way the database behind things is set up :)